supply/lib/supply/uploader.rb

```diff
@@ -152,11 +152,17 @@ def upload_mapping(apk_version_codes)
     end
 
     def upload_bundles
-      aab_path = Supply.config[:aab]
-      return [] unless aab_path
+      aab_paths = [Supply.config[:aab]] unless (aab_paths = Supply.config[:aab_paths])
```

Comment:
Ah, one more thing! There is a very small possibility that `Supply.config[:aab]` might be `nil` so `aab_paths` could be `[nil]`. Can we add a `aab_paths.compact!` after this line? Thhheeennnn I think we should be good 💪

Reply:
wouldn't be better to use `compact` instead of `compact!`? `compact!` would return nil if no removal were made.